django_freeradius/api/views.py

```diff
@@ -186,6 +190,20 @@ def create(self, request, *args, **kwargs):
         else:
             raise ValidationError(serializer.errors)
 
+    def perform_create(self, serializer):
+        user_model = get_user_model()
+        if app_settings.AUTOMATIC_GROUPNAME_ASSIGNMENT_RADIUSACCOUNTING:
+            try:
+                user = user_model.objects.get(username=serializer.validated_data.get('username'))
+                highest_priority_usergroup = return_highest_priority_usergroup(user)
```

Comment:
this is overkill, try just the following:

```python
group = user.radiususergroup_set.order_by('priority').first()
```

get rid of the function in `utils.py`

Reply:
I'll do this.